tuxemon/db.py

```diff
@@ -331,19 +331,23 @@ class MonsterEvolutionItemModel(BaseModel):
     tech: Optional[str] = Field(None, description="Technique parameter.")
 
     @field_validator("tech")
-    def technique_exists(cls: MonsterEvolutionItemModel, v: Any) -> Any:
+    def technique_exists(
+        cls: MonsterEvolutionItemModel, v: Optional[str]
+    ) -> Optional[str]:
+        if not v:
```

Comment:
I'm not sure why we are considering the `None` case here (it isn't in the other similar methods), but anyway, the `if` statements can be merged like this:

```py
if not v or has.db_entry("technique", v):
    return v
```

Reply:
Thanks, you're right. I missed item too (below), it's not similar because it involves evolution (eg.  botbot)
```
        {
            "path": "item", -> mandatory field, it evolves through item
            "at_level": 0, -> mandatory field, no minimum level
            "monster_slug": "av8r", -> mandatory field -> it evolves into av8r
            "item": "booster_tech" -> optional field regarding the item needed, that's why the field is default None
        }
```